kedro/extras/extensions/ipython.py

```diff
@@ -95,12 +92,11 @@ def load_ipython_extension(ipython):
 
     default_project_path = _find_kedro_project(Path.cwd())
 
-    try:
-        reload_kedro(default_project_path)
-    except (ImportError, ModuleNotFoundError):
-        logger.error("Kedro appears not to be installed in your current environment.")
```

Comment:
It makes sense that these have been removed, but I was just wondering if the user still gets a useful message if kedro actually isn't installed?

Reply:
Ah yes that's great!